ex2_system/include/leop/leop.h

```diff
@@ -31,8 +31,8 @@
 #include "deployablescontrol.h"
 #include "leop_eeprom.h"
 
-#define MAX_RETRIES 3
-bool execute_leop();
-bool deploy_all_deployables();
+#define MAX_RETRIES 5
```

Comment:
This should be _ATTEMPTS not _RETRIES.

Reply:
https://github.com/AlbertaSat/ex2_obc_software/pull/239/commits/a3ffc4a71e83611481cfc3fb4b3a14cb1eaaa176